aioauth/models.py

```diff
@@ -7,15 +7,16 @@
 
 ----
 """
-
+import dataclasses
```

Comment:
I think it looks shorter:

```suggestion
from dataclasses import dataclass
```

the same with other files